src/api.ts

```diff
@@ -4,29 +4,42 @@ export interface Trending  {
   count: number;
   document_type: string;
   title: string;
-  rating: IncompleteRating | CompleteRating
+  rating: Rating
 }
 
-type IncompleteRating = {
+
+// I thought it might make sense to consolidate the typing by collapsing bothe the incompleteRating and CompleteRating into a single Rating with a new 'complete' field for indexing
+type Rating = {
```

Comment:
This works, also see optional type parameters: https://www.typescripttutorial.net/typescript-tutorial/typescript-optional-parameters/